community/kernel-api/src/main/java/org/neo4j/internal/kernel/api/ExecutionStatistics.java

```diff
@@ -17,17 +17,22 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package org.neo4j.kernel.api;
-
-import org.neo4j.io.pagecache.tracing.cursor.PageCursorTracer;
+package org.neo4j.internal.kernel.api;
 
 /**
- * Defines methods that expose various statistical information from kernel
+ * Expose statistical information about the current transaction.
  */
-public interface ExecutionStatisticsOperations
+public interface ExecutionStatistics
 {
     /**
-     * @return provide current page cursor tracer that expose current transaction page cache statistic
+     * Returns the number of page hits in the current transaction
+     * @return the number of page hits in the current transaction
+     */
+    long pageHits();
```

Comment:
Now when its a transaction that is ExecutionStatistics this java doc is not really true since it returns hits and faults of current thread local tracer and for example for rest and bolt transactions that's not true since those will be rebound to different threads

Reply:
Do you have any suggestion it used to say `expose current transaction page cache statistic`

Reply:
yes i think just update javadoc that its expose current counters without any promises that those are that particular transaction counters